server/db/models/review.js

```diff
@@ -0,0 +1,18 @@
+var Sequelize = require('sequelize');
+
+var db = require('../_db');
+
+module.exports = db.define('review', {
+  stars: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+    validate: {
+      min: 1,
+      max: 5
+    }
+  },
+  content: {
+    type: Sequelize.TEXT,
+    validate: {len: 20}
+  }
+},{})
```

Comment:
Given that the second parameter in the `db.define` is optional, and you're not using it, it is cleaner to simply omit the empty object.